en_ewt-ud-test.conllu

```diff
@@ -9452,15 +9452,15 @@
 10	in	in	ADP	IN	_	11	case	11:case	_
 11	September	September	PROPN	NNP	Number=Sing	4	obl	4:obl:in	_
 12	of	of	ADP	IN	_	13	case	13:case	_
-13	'67	'67	NUM	CD	NumType=Card	11	nmod	11:nmod:of	SpaceAfter=No
+13	'67	'67	NUM	CD	Abbr=Yes|NumForm=Digit|NumType=Card	11	nmod	11:nmod:of	SpaceAfter=No
```

Comment:
I hadn't thought to use `Abbr=Yes` for these. Usually `Abbr=Yes` implies that the lemma disambiguates to the full form, right? Is it OK to make an exception for years?

If this is `Abbr=Yes`, ones without a century-eliding apostrophe probably should be as well, e.g. in the sentence that mentions "memorabilia from the 70's and 80's".

Reply:
IDK, but I'm happy to remove that and leave it as an issue for later

https://github.com/UniversalDependencies/UD_English-EWT/pull/336/commits/77bf47661d92433f3a408d30c76feb4489391b18

Reply:
I could see a case for lemmatizing to 1967, but we could also say that belongs in a TimeML-like layer which disambiguatess dates.

Reply:
Yeah that seems to be too semantic for a lemma. If we establish that `Abbr=Yes` does not require an expanded lemma in cases like "etc.", then I would be on board with saying that abbreviated years with an apostrophe can be lemmas. But we should also add in missing apostrophes (70's => '70 Number=Plur).